design/ceph/object/ceph-rgw-k8s-sa-authentication.md

```diff
@@ -0,0 +1,143 @@
+---
+Service authentication with vault for RGW
+target-version: release-1.14
```

Comment:
You're planning on implementing it immediately for the release soon?

Reply:
Yes, thats the plan. The code changes for this implementation are pretty minimal. I will push the PR along with the design PR